server.go

```diff
@@ -260,6 +301,11 @@ func loadState() (*grange.State, int) {
 	return &state, warnings
 }
 
+func cleanupStatsd() {
+	stats.Close()
+	stats = &noopStatsd{}
```

Comment:
Imagine I'm setting it to `nil` instead, which would be more normal in a cleanup routine I think.